doc/user/flycheck-versus-flymake.rst

```diff
@@ -429,13 +316,13 @@ syntax checks as there are visible windows in the current Emacs session.
 Checking for changes
 ~~~~~~~~~~~~~~~~~~~~
 
-**Flymake** uses a *separate* timer (in ``flymake-timer``) to periodically check
+**Flymake** uses a *separate* timer (in `flymake-timer`) to periodically check
```

Comment:
Hmmm, I don't think so, but I'll have to check,

Reply:
I just saw that timer was still around in the code, but I didn't go through each line to figure out if the code was behaving as described.  I was hoping you would be more experienced with the code base than I am ;)

Reply:
The idle timer after change dance is just the same in Flycheck and Flymake. It's just that Flymake uses a local variable. But it's not really checking "periodically"